apps/src/lib/node/ledger/ethereum_node/events.rs

```diff
@@ -750,6 +783,45 @@ pub mod eth_events {
     mod test_events {
         use super::*;
 
+        #[test]
+        fn test_transfer_to_namada_decode() {
+            let data: Vec<u8> = vec![
```

Comment:
This event data was copied straight from ledger logs while I was testing out decoding an event from the real smart contracts. This might be better as an example rather than a test.